### ISSUE TITLE
Not working with build optimizer enabled on Angular CLI

### ISSUE THREAD

**Issue by wceolin** [KillerCodeMonkey/ngx-quill#102]
The latest version of the library isn't working when enabling the `build-optimizer` flag using the Angular CLI (1.6.4). I get an `Uncaught TypeError: Cannot read property 'call' of undefined` error.

*Steps to set up and reproduce*

```
ng new quill --routing
yarn add ngx-quill

Import QuillModule in the app.module

ng build --env=prod --aot --build-optimizer
```

Then, try opening the dist folder. The error above will be logged into the console. Maybe related to #74?

**Comment by KillerCodeMonkey** [comment]
i will check this ... hopefully this week. Or you can try to solve it

**Comment by patrick-nurt** [comment]
Has anyone managed to solve this one?

**Comment by KillerCodeMonkey** [comment]
there is an issue in the angular cli repo: https://github.com/angular/angular-cli/issues/6196

and check the first answer here:
https://stackoverflow.com/questions/46720805/cannot-read-property-call-of-undefined-angular-2-build-error-potential-solu

hopefully that helps.

**Comment by wceolin** [comment]
It should have been fixed with `@angular-devkit/build-optimizer@0.0.27` but, apparently, it wasn't completely (as this bug showed up in the latest version). I didn't have time to dig into it, though. I'll try to have a better look next week.

**Comment by wceolin** [comment]
I was having a look at this issue. If we import `quill` from the minified version (dist/quill.min.js), then it works fine (temporarily change the `main` entry in quill's package.json file to `"main": "dist/quill.min.js"` and it will work.

I'm not sure if it's a `quill` or a `build-optimizer` problem, though. I'm closing this issue as it's not related to `ngx-quill`.
